src/pages/loading-waiting-states/progress-bar/index.tsx

```diff
@@ -0,0 +1,234 @@
+            </AppBar>
+            <div className={classes.container}>
+                <div className={classes.uploadButtonContainer}>
+                    <Button
+                        variant={'contained'}
+                        color={'primary'}
+                        startIcon={<VerticalAlignTop />}
+                        onClick={uploadFile}
+                    >
+                        <Typography noWrap color={'inherit'}>
+                            UPLOAD NEW FILE
+                        </Typography>
+                    </Button>
+                </div>
+                <Card>
+                    <FormControl className={classes.formControlStyle} component="fieldset">
+                        <RadioGroup aria-label="folder" name="folder" value="1">
+                            {foldersList.map((option, i) => (
+                                <FormControlLabel
+                                    className={classes.formLabel}
+                                    key={i}
+                                    value={option.value}
+                                    control={<Radio />}
+                                    label={
+                                        <div className={classes.radioLabel}>
+                                            <Folder className={classes.iconStyle} />
+                                            <Typography> {option.label} </Typography>
+                                        </div>
+                                    }
+                                />
+                            ))}
+                        </RadioGroup>
+                    </FormControl>
+                </Card>
+                <List data-cy={'list-content'} disablePadding component="nav" className={classes.placementOfList}>
+                    {list.map(
+                        (item, i): JSX.Element => (
+                            <div key={`itemKey${i}`} className={classes.infoListItemStyle}>
+                                <InfoListItem
+                                    classes={{
+                                        subtitle: classes.subTitle,
+                                    }}
+                                    key={i}
+                                    hidePadding
+                                    ripple
+                                    backgroundColor={Colors.black[900]}
+                                    fontColor={Colors.black[50]}
+                                    color={Colors.black[50]}
+                                    title={item.name}
+                                    subtitle={item.status}
+                                    icon={<Description />}
+                                    iconAlign="left"
+                                    iconColor={Colors.black[200]}
+                                    rightComponent={
+                                        <Button variant="outlined" className={classes.closeButtonContainer}>
```

Comment:
I expect the cancel button to dismiss the snackbar?

And "Cancel" will change to "View" once the upload is complete?

Reply:
I think this is still TODO